config/initializers/general_metrics.rb

```diff
@@ -0,0 +1,17 @@
+require 'metrics_adapter/rails'
```

Comment:
can we change the file name to `metrics_adapter`?

Reply:
Done!